bundles/net.tourbook/src/net/tourbook/map25/ui/SlideoutMap25_MapOptions.java

```diff
@@ -35,14 +43,6 @@
 import org.eclipse.swt.widgets.Spinner;
 import org.eclipse.swt.widgets.ToolBar;
 
-import net.tourbook.Messages;
-import net.tourbook.common.UI;
-import net.tourbook.common.font.MTFont;
-import net.tourbook.common.tooltip.ToolbarSlideout;
-import net.tourbook.map25.Map25App;
-import net.tourbook.map25.Map25ConfigManager;
-import net.tourbook.map25.Map25View;
-
 /**
```

Comment:
Blue box is OK, I marked it because I thought it is not working but it is.

Red box: 
* Text key is displayed
* Cannot see a difference when checked or unchecked

![mt photo](https://user-images.githubusercontent.com/1283445/113831745-3bd39300-9788-11eb-9105-c5431f8b0b14.png)

Reply:
Wolfgang, did you remember this problem:
https://github.com/wolfgang-ch/mytourbook/commit/d3e09e3afe3c2b6d439ebd9c85afe1603eeb909b#commitcomment-41469693

There you removed the scaling part. but that looks not good on my two W10 laptops and in my VMWare Linux.
Not sure where this depends on.

With scaling on, it looks good everywhere but shows more used resources in "sleak"
with scaling off, it is like in the last version.

![grafik](https://user-images.githubusercontent.com/22706487/113892207-14032000-97c6-11eb-9f13-c7b7d958a614.png)

Reply:
> 
> 
> Wolfgang, did you remember this problem:
> [d3e09e3#commitcomment-41469693](https://github.com/wolfgang-ch/mytourbook/commit/d3e09e3afe3c2b6d439ebd9c85afe1603eeb909b#commitcomment-41469693)
> 
> There you removed the scaling part. but that looks not good on my two W10 laptops and in my VMWare Linux.
> Not sure where this depends on.
> 
> With scaling on, it looks good everywhere but shows more used resources in "sleak"
> with scaling off, it is like in the last version.
> 
> ![grafik](https://user-images.githubusercontent.com/22706487/113892207-14032000-97c6-11eb-9f13-c7b7d958a614.png)

I tried to understand the photo painting, but giving up for now, why it isn't painted with a larger (not scaled) image.

Can you set the missing text, afterwards I will merge it.

Reply:
yes of coarse.

should i call it :
"scaled photos"
or better
"force scaled photos"

because sometimes, like in you're case its always scaled.

should i add this line only to messages.properties and Messages.java or also in messages_de.properties?

Reply:
We should not call it scaled, because scaled could be larger OR smaller.

Also, I think we should look over the "Tellerrand" (don't know if there is an english word for it) and the user should be able to set/select the image size.

There may be fixed sizes, e.g. 160, 300, ... and a variable size.

When you are not yet implementing this sizing feature, then the checkbox label could be "Small photo" which you can also translate into german (very difficult task :-)

Reply:
the easiest for me is to copy youre hillshading code for maps options

![grafik](https://user-images.githubusercontent.com/22706487/114073930-22276e00-98a4-11eb-980e-f9aaee4eaa92.png)

Reply:
checkbox and text is not ready yet

Reply:
like this:
![grafik](https://user-images.githubusercontent.com/22706487/114074693-f35dc780-98a4-11eb-8748-1fda16e3bbdc.png)

Reply:
This is OK